docs/volumes.adoc

```diff
@@ -0,0 +1,78 @@
+// vim: tw=79
+= Gluster Volume
+:toc:
+
+== List Volumes
+
+Sample Request
+
+----------
+curl -XGET http://127.0.0.1/api/1.0/5291c055-70d3-4450-9769-2f6fd4932afb/GetVol
+umeList
+----------
+
+Sample Response
+
+----------
+Status: 200 OK
+
+[{
+	"cluster_id": "5291c055-70d3-4450-9769-2f6fd4932afb",
+	"deleted": "",
+	"name": "vol1",
+	"status": "Stopped",
+	"vol_id": "80e98588-176c-4220-b41c-e95cd7137aff",
+	"vol_type": "Distribute",
+	"mount_opts": "",
+	"path": "tendrl.lab.example:/root/bricks/vol1_b2",
+	"port": "49152",
+	"fs_type": "",
+	"hostname": "tendrl.lab.example",
+	"brick_count": "2"
+}, {
+	"name": "Volume_009",
+	"status": "None",
+	"vol_id": "b0d4daa8-2ba6-4ea4-b510-c486c1b62e9f",
+	"vol_type": "Distribute",
+	"hostname": "tendrl.lab.example",
+	"mount_opts": "",
+	"path": "tendrl.lab.example:/root/bricks/vol9_b1",
+	"port": "None",
+	"cluster_id": "5291c055-70d3-4450-9769-2f6fd4932afb",
+	"fs_type": "None",
+	"brick_count": "1",
+	"deleted": ""
+}]
+----------
+
+== Create Volume
+
+Assuming you have pre-existing bricks, we can create a new volume in a specific cluster.
+
+Sample Request
+----------
+curl -XPOST -d '{"Volume.volname":"Volume_009","Volume.bricks":["dhcp-1.lab.ten
+drl.example:/root/bricks/vol9_b1"]}' http://127.0.0.1/api/1.0/5291c055-70d3-4450-9769-2f6fd4932afb/GlusterCreateVolume
```

Comment:
Maybe it would be better to add example with more bricks and show how to specify how to create replicated/striped volumes if they are supported. Volume with one brick is a little misleading.